File: CarRental.Web/Binding/Models/ReservationModel.js

(function (cr) {
    var ReservationModel = function (reservationId, car, rentalDate, returnDate) {

        var self = this;

        self.ReservationId = ko.observable(reservationId);
        self.Car = ko.observable(car);
        self.RentalDate = ko.observable(rentalDate);
        self.ReturnDate = ko.observable(returnDate);

        self.CancelRequest = ko.observable(false);
    }
    cr.ReservationModel = ReservationModel;
}(window.CarRental));

File: CarRental.Web/Models/ReservationModel.cs
using CarRental.Client.Entities;
using System;
using System.Collections.Generic;
using System.Linq;

namespace CarRental.Web.Models
{
    public class ReservationModel
    {
        public int Car { get; set; }
        public DateTime PickupDate { get; set; }
        public DateTime ReturnDate { get; set; }
    }
}
